internal/admin/dashboard/templates/index.html

```
@@ -253,6 +260,160 @@ <h2>User Settings</h2>
+        <section class="settings-panel settings-guardrails-list">
+            <div class="aliases-editor-header">
+                <div>
+                    <h3>Instances</h3>
+                    <p class="alias-form-hint">Each instance has a reusable name, a type, an optional user path for future UI visibility scoping, and a JSON-backed config payload for that type.</p>
+                </div>
+                <button type="button" class="pagination-btn pagination-btn-primary" :disabled="guardrailTypesLoading || guardrailFormSubmitting || !guardrailsAvailable" @click="openGuardrailCreate()">Create Guardrail</button>
+            </div>
+
+            <div class="table-toolbar" x-show="guardrailsAvailable">
+                <div class="table-toolbar-main">
+                    <input type="text" class="filter-input" placeholder="Filter by name, type, user path, summary..." x-model="guardrailFilter">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)